setup.py

```diff
@@ -11,6 +11,10 @@
 with open('requirements.txt') as f:
     requirements = f.read().splitlines()
 
+requirements_dev = []
+with open('requirements.txt') as f:
+    requirements = f.read().splitlines()
```

Comment:
I suppose this was meant to read `requirements_dev`?

Reply:
I have literally no idea what you are talking about :-)